src/headline_grabber/pipelines/__init__.py

```diff
@@ -16,6 +17,7 @@
         ClassifySubject(),
         FilterMaxResults(),
         ScoreSentiment(),
+        FilterSentiment(),
```

Comment:
@mralderman Please move the filter sentiment to above score sentiment.

Reply:
In order to filter out the sentiments, doesn't it have to set them first? The reason it follows is because I was under the impression that ScoreSentiment set the sentiment and then the filter can apply. I could be wrong on this though.

Reply:
Yes @mralderman is correct, @tonthattuit 's request makes no sense